lms/static/sass/views/_verification.scss

```diff
@@ -821,6 +829,114 @@
 
     // indiv slides - review
     #wrapper-review {
+      color: $black;
+
+      .page-title {
+          border-bottom: 2px solid $darkGrey;
```

Comment:
use $gray-d_ to match darkGrey (we are trying to avoid using the direct / older color variables when possible.)